app/waf/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "module": "commonjs",
     "target": "es5",
     "outDir": "./dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "experimentalDecorators": true
```

Comment:
Why we needn't this when we implement hasMany for pool/member?

Reply:
same question. why do we need it? The code seems run/test well without it.

Reply:
it will show some experimental decorator error on my VS, if not add these

Reply:
I tried `npm start` `npm test` `npm run build`. There's no error happens.